Example/AppDelegate.swift

```diff
@@ -28,6 +28,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             return MiniApp.MAOrientationLock
         }
     }
+
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        guard let components = URLComponents(url: url, resolvingAgainstBaseURL: true), let host = components.host else {
+            return false
+        }
+        print("Host: ", host)
```

Comment:
do you think it will be useful to print the host in our future developments? (you can keep the log, I'm just curious)

Reply:
no no. This is just to print the logs for now, we will be removing them when we actually implement QR functionality